lib/stdlib/Duden/Listen.ddp

```diff
@@ -221,6 +223,34 @@ Die öffentliche Funktion Produkt_Zahlen_Liste mit dem Parameter liste vom Typ Z
 Und kann so benutzt werden:
 	"das Produkt aller Zahlen in <liste>"
 
+[
+	Gibt die kleinste Zahl in der gegebenen Zahlen Liste zurück.
+	l := {e1, e2, e3, ..., en}
+	f(l) = min(e1, e2, e3, ..., en)
+	f({}) = 0
+]
+Die öffentliche Funktion Minimum_Zahlen_Liste mit dem Parameter liste vom Typ Zahlen Liste, gibt eine Zahl zurück, macht:
+	Wenn liste leer ist, gib 0 zurück.
+	Die Zahl min ist der maximale Wert einer Zahl.
+	Für jede Zahl z in liste, wenn z kleiner als min ist, speichere z in min.
+	Gib min zurück.
+Und kann so benutzt werden:
+	"die kleinste Zahl in <liste>"
+
+[
+	Gibt die größte Zahl in der gegebenen Zahlen Liste zurück.
+	l := {e1, e2, e3, ..., en}
+	f(l) = max(e1, e2, e3, ..., en)
+	f({}) = 0
+]
+Die öffentliche Funktion Maximum_Zahlen_Liste mit dem Parameter liste vom Typ Zahlen Liste, gibt eine Zahl zurück, macht:
+	Wenn liste leer ist, gib 0 zurück.
+	Die Zahl max ist der minimale Wert einer Zahl.
+	Für jede Zahl z in liste, wenn z größer als max ist, speichere z in max.
+	Gib max zurück.
+Und kann so benutzt werden:
+	"die größte Zahl in <liste>"
+
```

Comment:
Diese Funktionen existieren schon in Duden/Statistik. https://doku.ddp.im/Programmierung/Standardbibliothek/Statistik/